rs/rosetta-api/icrc1/tests/golden_state_upgrade_downgrade.rs

```diff
@@ -11,9 +11,9 @@ mod common;
 
 #[cfg(not(feature = "u256-tokens"))]
 #[test]
-fn should_upgrade_icrc_ck_btc_canister_with_golden_state() {
-    const CK_BTC_LEDGER_CANISTER_ID: &str = "mxzaz-hqaaa-aaaar-qaada-cai";
-    const CK_BTC_LEDGER_CANISTER_NAME: &str = "ckBTC";
+fn should_upgrade_icrc_ck_btc_canisters_with_golden_state() {
+    // const CK_TEST_BTC_LEDGER: (&str, &str) = ("mc6ru-gyaaa-aaaar-qaaaq-cai", "ckTestBTC");
```

Comment:
Is this on purpose?

Reply:
It's related to the `TODO: FI-1471` further down, but I could remove the TODOs from this PR and just track the necessary changes in that ticket.